src/containers/VisitPage.js

```diff
@@ -62,37 +52,19 @@ class VisitPage extends Component {
       userChanged:           {},
       snippets:              props.snippets,
     };  // end this.state {}
+  };  // End constructor()
 
-    this.steps = [
-      {
-        form:              CurrentBenefitsStep,
-        key:               'currentBenefits',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              HouseholdStep,
-        key:               'household',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              CurrentIncomeStep,
-        key:               'currentIncome',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              CurrentExpensesStep,
-        key:               'currentExpenses',
-        updateClientValue: this.changeCurrent,
-      },
-      {
-        form:              PredictionsStep,
-        key:               'predictions',
-        updateClientValue: this.changeFuture,
-      },//,
-    //  { title: 'Graphs', form: ResultsGraph }
-    ];  // end this.steps {}
+  componentDidMount() {
+    // If the user has refreshed the page while they were on a later step, then we will
+    // go back to that step, but since the client data is currently stored in memory, the
+    // client data will be lost. So if we are loading on a step that is not the first step,
+    // go back to the first step to start again
+    const firstStepURL = `${this.getPathPrefix()}/${STEPS[ 0 ].key}`;
 
-  };  // End constructor()
+    if (this.props.history.location !== firstStepURL) {
```

Comment:
Is this needed? If you're on the first step, you can still go to the first step without difficulty. Is it heavy to replace history and navigate to a step? Are there other reasons to avoid it?

Reply:
I had some issues with React Router throwing errors to the console when transitioning to the same page.

Reply:
Ah, got it! A comment to that effect would probably be useful to someone trying to tighten up the code in the future.